.github/workflows/a-team.yml

```diff
@@ -19,6 +19,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Create yum repo
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_BUCKET: ${{ secrets.AWS_BUCKET }}
```

Comment:
I propose we use a more verbose secret, the local env for this action run is fine, but we want to differentiate between the various types of buckets we will have on AWS in our GH secret store, i.e for your use it would be `AWS_BUCKET: ${{ secrets.AWS_BUCKET_YUM }}` to indicate you want to use the yum repo bucket, and another secret for Guy's needs e.g `${{ secrets.AWS_BUCKET_IMG}}`, Which are secrets that I would hope terraform can somehow populate (securely) , I'm still figuring it out on the terraform end though.

Reply:
Good idea.